src/pages/workspaces/workspace/jobHistory/SubmissionDetails.js

```diff
@@ -160,7 +160,7 @@ const SubmissionDetails = _.flow(
       div({ style: { display: 'grid', gridTemplateColumns: '1fr 4fr' } }, [
         div({ style: { display: 'grid', gridTemplateRows: '1fr auto' } }, [
           makeSection('Workflow Statuses',
-            [statusType.succeeded.id, statusType.failed.id, statusType.running.id, statusType.submitted.id].filter(s => statusGroups[s]).map(
+            ['succeeded', 'failed', 'running', 'submitted'].filter(s => statusGroups[s]).map(
```

Comment:
This is not actually necessary to fix the bug, but I think it makes it more obvious what is going on. We are accessing  statusGroups.succeeded by doing statusGroups['succeeded'].

Reply:
I think I would prefer something like an enumerated type here as well. But I am ok with it as-is for this PR